dev/extra-ml-requirements.txt

```diff
@@ -2,15 +2,15 @@
 ## must install in order to use various MLflow Python modules.
 ##
 # Required by mlflow.azureml
-azureml-sdk; python_version >= "3.0"
+azureml-sdk==1.2.0; python_version >= "3.0"
```

Comment:
Can we remove the python requirement since we don't support < 3.5 anymore?

Reply:
Done!